### DISCUSSION TITLE
MySQL Transaction

### DISCUSSION THREAD

**Discussion by ghost** [eveningkid/denodb#196] [Q&A] [answered]
Hi!

I've run thru your codes and found out that it is possible to implement a transaction in MySQLConnector. 

![Screenshot_2021_0128_125501](https://user-images.githubusercontent.com/69180862/106091847-2b8bb000-6168-11eb-9f4d-0b392a97f57f.png)

Right now I'm not really sure how to use it, like properly implement it. Should I just access it via MySQLConnector.transaction? Your docs page doesn't seem to have a documentation about it.  

PS, I really love the lib and am looking forward for your future releases.

**Comment by eveningkid** [comment, marked as answer]
Hey,

Sorry for the long wait, I was away for some time.

You are very much right about this, and I answered today in #204 about it.

Now I see there is more than one person looking for this so I'll see what I can do!

This has been an issue since other database plugins don't offer transactions at the moment but I could make it a MySQL-only for the time being.

I will reopen the issue, you can subscribe for updates when they'll come :)

Good day

**Reply by eveningkid** [reply]
Transactions have just been added and you can start using them from today :)
[Read more on the documentation.](http://eveningkid.com/denodb-docs/docs/guides/transactions)

Have a great day!

**Reply by ghost** [reply]
Thanks, man. Really appreciate it. Gonna look into it if I have time.

**Reply by eveningkid** [reply]
Sure thing, you have all the time you need :)
